lib/utils/snackbar.dart

```diff
@@ -0,0 +1 @@
+
```

Comment:
this is empity?

Reply:
Oh sorry, In this file I wanted to add the snackbar which is causing error in the `home_view.dart`.